### DISCUSSION TITLE
fastapi mail status error 421,

### DISCUSSION THREAD

**Discussion by Okafor-Ifeanyi** [fastapi/fastapi#8495] [Questions] [answered]
### First Check

- [X] I added a very descriptive title to this issue.
- [X] I used the GitHub search to find a similar issue and didn't find it.
- [X] I searched the FastAPI documentation, with the integrated search.
- [X] I already searched in Google "How to X in FastAPI" and didn't find any information.
- [X] I already read and followed all the tutorial in the docs and didn't find an answer.
- [X] I already checked if it is not related to FastAPI but to [Pydantic](https://github.com/samuelcolvin/pydantic).
- [X] I already checked if it is not related to FastAPI but to [Swagger UI](https://github.com/swagger-api/swagger-ui).
- [X] I already checked if it is not related to FastAPI but to [ReDoc](https://github.com/Redocly/redoc).

### Commit to Help

- [X] I commit to help with one of those options 👆

### Example Code

```python
from fastapi import APIRouter
from starlette.responses import JSONResponse
from fastapi_mail import FastMail, MessageSchema, ConnectionConfig


router = APIRouter(
    tags=["Authentication"]
)

# Mail configuration for sending request
conf = ConnectionConfig(
    MAIL_USERNAME= settings.mail_username,
    MAIL_PASSWORD= settings.mail_password,
    MAIL_FROM= settings.mail_from,
    MAIL_PORT= settings.mail_port,
    MAIL_SERVER= settings.mail_server,
    MAIL_TLS= settings.mail_tls,
    MAIL_SSL= settings.mail_ssl,
    USE_CREDENTIALS= settings.use_credentials,
)

# link = print(router.url_path_for('request_reset'))
link = print("//https")

html = f"""
<p> Hi current_user /n /n Kindly click here {link} to reset your password /n /n Thanks /n UU81</p> 
"""


async def send_reset_email(user) -> JSONResponse:
    message = MessageSchema(
        subject="UNIQUE UNIPORT 81",
        recipients=user.dict().get("email"),  # List of recipients, as many as you can pass 
        body=html,
        subtype="html"
        )
    fm = FastMail(conf)
    print(conf.MAIL_FROM, conf.MAIL_PASSWORD)
    await asyncio.sleep(fm.send_message(message))
    
    return JSONResponse(status_code=200, content={"message": "email has been sent"})
```

### Description

* This an event wbeb app
* The user inherits an email from the request password router 
* My wish is to send an email to the users email wilt the link to the reset password router
* i get a status error of 421 each time I run just mail code
* Inside my webapp i get "RuntimeWarning: Enable tracemalloc to get the object allocation traceback"
* In full      """  C:\Users\HP\Dropbox\My PC (UAMACCOUNTS)\Desktop\knight\.\app\routers\auth.py:82: RuntimeWarning: coroutine 'send_reset_email' was never awaited                                                                                                                                     aited
  send_reset_email(user)
RuntimeWarning: Enable tracemalloc to get the object allocation traceback """
* Please Help

### Operating System

Windows

### Operating System Details

HP Laptop 15-da0xxx
Processor	Intel(R) Core(TM) i5-8250U CPU @ 1.60GHz   1.80 GHz
Installed RAM	8.00 GB
Device ID	10D2CA33-3355-413F-998D-E703BC4195B1
Product ID	00325-81250-88248-AAOEM
System type	64-bit operating system, x64-based processor
Pen and touch	No pen or touch input is available for this display

Windows Specification
Edition	Windows 10 Home
Version	21H2
Installed on	‎25/‎12/‎2021
OS build	19044.1645
Experience	Windows Feature Experience Pack 120.2212.4170.0

### FastAPI Version

0.75.2

### Python Version

Python 3.8.10

### Additional Context

Please Help

**Comment by narmatov-asylbek** [comment, marked as answer]
@Okafor-Ifeanyi The traceback shows that the coroutine is not awaited. This is not surprising, as FastMail requires async syntax. You can check it from their documentation: https://sabuhish.github.io/fastapi-mail/.

In your code, you should use async syntax. You should transform your code fragment from:
```python
fm = FastMail(conf)
await asyncio.sleep(fm.send_message(message))
```

To 

```python
fm = FastMail(conf)
await fm.send_message(message)

```
